tasks/TakeOutGarbage.tex

```diff
@@ -0,0 +1,61 @@
+\section{Take Out the Garbage [House Holder]}
+The robot has to take out the trash, recycling and compost.
+
+% \subsection{Focus}
+% This test focuses on object perception, manipulation, and planning.
+
+\subsection{Main Goal}
+All garbage in the apartment has has been deposited into the correct exterior bin.
+
+\noindent\textbf{Reward:} 450pts (150pts per type of garbage)
```

Comment:
Not sure if 150pts is enough reward for taking out a plastic bag from a bucket on the floor. It has never been done before.

Just 150pts for taking out, opening a lid, and placing inside is completely out of the question. Not even I am that cheap with scoring.

Reply:
Bumped to 300pts for taking a single bag out and putting it on a designated spot on the floor somewhere in the arena.